elasticsearch/spec/unit/elasticsearch_product_validation_spec.rb

```diff
@@ -35,13 +35,13 @@
     { 'content-type' => 'json' }
   end
 
-  def error_requests_and_expectations
-    expect { client.count }.to raise_error Elasticsearch::NotElasticsearchError
+  def error_requests_and_expectations(message = Elasticsearch::NOT_ELASTICSEARCH_WARNING)
```

Comment:
Just verifying there's a test case for "unsupported product" and "unsupported distribution"?

Reply:
Yes, since I was using the same error class, I expected it to raise the same object with the two different messages in the tests (I pass in the other error message in the expectations for the 'unsupported distribution' tests). But since using an `UnsupportedProductError` class is more explicit, I'll update the PR to expect this other error type :+1: